actionview/lib/action_view/helpers/sanitize_helper.rb

```diff
@@ -10,7 +10,7 @@ module Helpers #:nodoc:
     # These helper methods extend Action View making them callable within your template files.
     module SanitizeHelper
       extend ActiveSupport::Concern
-      # Sanitizes HTML input, stripping all tags and attributes that aren't whitelisted.
+      # Sanitizes HTML input, stripping all tags and attributes that aren't permitted.
```

Comment:
The below use of "permitted" is good, but this one feels a bit weak. How about "stripping all but known-safe tags and attributes"?